README.md

```diff
@@ -469,6 +469,15 @@ There are 3 main table migration workflows, targeting different table types. All
 
 [[back to top](#databricks-labs-ucx)]
 
+## Workflow Linter Workflow
+
+Please note that this is an experimental workflow.
+
+The `experimental-workflow-linter` workflow lints accessible code belonging to all workflows/jobs present in the
+workspace. The linting emits problems indicating what to resolve for making the code Unity Catalog compatible.
```

Comment:
put an image to docs folder and place it here in the doc, so that people would know what this workflow means by the example output.